zerobuf/Zerobuf.cpp

```diff
@@ -30,12 +30,12 @@ Zerobuf::Zerobuf( AllocatorPtr alloc )
 
 Zerobuf::Zerobuf( Zerobuf&& rhs )
 {
-    rhs.notifyChanging();
     _allocator = std::move( rhs._allocator );
     rhs._allocator.reset( new NonMovingAllocator( rhs.getZerobufStaticSize(),
                                                  rhs.getZerobufNumDynamics( )));
     uint32_t& version = rhs._allocator->getItem< uint32_t >( 0 );
     version = ZEROBUF_VERSION_ABI;
+    rhs.notifyUpdated();
```

Comment:
Right, this is one of the questions I have about the code (I just moved down the notify call). That's why I had not assigned anyone yet...